embassy-usb/src/descriptor.rs

```diff
@@ -326,11 +326,11 @@ pub(crate) fn device_descriptor(config: &Config) -> [u8; 18] {
         18,   // bLength
         0x01, // bDescriptorType
         0x10,
-        0x02,                     // bcdUSB 2.1
-        config.device_class,      // bDeviceClass
-        config.device_sub_class,  // bDeviceSubClass
-        config.device_protocol,   // bDeviceProtocol
-        config.max_packet_size_0, // bMaxPacketSize0
+        (config.bcd_usb >> 8) as u8, // bcdUSB
```

Comment:
the `0x10` above is also part of `bcd_usb`, it's the lower byte. for example USB 2.1 is `0x0210`.

The config default should also be `0x0210` instead of `0x02`